app/services/moderator/merge_user.rb

```diff
@@ -49,24 +48,6 @@ def update_social
       end
     end
 
-    def remove_additional_email
-      return if @delete_user.email.blank?
-
-      email_attr = {
```

Comment:
I couldnt find a reason we updated these and then destroyed the user immediately after so I removed it. Please let me know if I missed something here!!!

Reply:
I went hunting around for something that might indicate these are used after a user is removed but didn't find anything. I think the key here is this line `@delete_user.unsubscribe_from_newsletters` to ensure they are removed from MailChimp